src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.Utf8JsonWriter.cs

```diff
@@ -12,6 +12,9 @@ public static partial class JsonSerializer
         /// <param name="writer">The writer to write.</param>
         /// <param name="value">The value to convert and write.</param>
         /// <param name="options">Options to control the behavior.</param>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="writer"/> is null.
```

Comment:
@carlossanlop - will your tool pick up changes like these and port to docs or do I have to do it manually?

The exception section of the Serialize APIs is a bit light on details currently:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.serialize?view=netcore-3.0

Reply:
Yes. The tool will pick it up.
The reason: This exception had not yet been added to the list of exceptions for [this Serialize overload](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.serialize?view=netcore-3.0#System_Text_Json_JsonSerializer_Serialize__1_System_Text_Json_Utf8JsonWriter___0_System_Text_Json_JsonSerializerOptions_). If it had been there already, the modification would've had to be done manually.

Reply:
A couple of requests:

1. Can you fix the spacing? In other words, put everything in one line?
2. Can you make the null word a see langword?

Like this:
```xml
<exception cref="ArgumentNullException"><paramref name="writer" /> is <see langword="null" />.</exception>
```

Reply:
> A couple of requests

Can we fix this on the docs side? I'll keep the feedback in mind for future updates. Thanks @carlossanlop!

Reply:
Now that I think of it, the tool will probably detect the extra spacing and fix it, and will detect the null word and convert it. Let's see if it works.

Reply:
@ahsonkhan I just realized another thing: [I don't see this overload ](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.serialize?view=netcore-3.0) in netcore-3.0 in Docs. I pasted the wrong one above (it has 3 parameters). This one has 4 parameters. Has it been sent to Maira via ref assemblies?

Reply:
> Has it been sent to Maira via ref assemblies?

Yes, it's there:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.serialize?view=netcore-3.0#System_Text_Json_JsonSerializer_Serialize_System_Text_Json_Utf8JsonWriter_System_Object_System_Type_System_Text_Json_JsonSerializerOptions_

We have a generic overload (with 3 args) and the non-generic overload (with 4 args) for serializing into a `Utf8JsonWriter`.